public/index.html

```diff
@@ -1267,6 +1267,37 @@ <h4 class="wide100p textAlignCenter">
+                                        </h4>
+                                        <div data-tg-type="mancer, ooba, koboldcpp, aphrodite, tabby" class="flex-container flexFlowRow gap10px flexShrink">
+                                            <div data-newbie-hidden class="alignitemscenter flex-container flexFlowColumn flexBasis48p flexGrow flexShrink gap0">
+                                                <small data-i18n="Allowed Length">Allowed Length</small>
+                                                <input class="neo-range-slider" type="range" id="dry_allowed_length_textgenerationwebui" name="volume" min="1" max="20" step="1" />
+                                                <input class="neo-range-input" type="number" min="1" max="20" step="1" data-for="dry_allowed_length_textgenerationwebui" id="dry_allowed_length_counter_textgenerationwebui">
+                                            </div>
+                                            <div data-tg-type="mancer, ooba, koboldcpp, aphrodite" data-newbie-hidden class="alignitemscenter flex-container flexFlowColumn flexBasis48p flexGrow flexShrink gap0">
+                                                <small data-i18n="Multiplier">Multiplier</small>
+                                                <input class="neo-range-slider" type="range" id="dry_multiplier_textgenerationwebui" name="volume" min="0" max="10" step="0.01" />
+                                                <input class="neo-range-input" type="number" min="0" max="5" step="0.01" data-for="dry_multiplier_textgenerationwebui" id="dry_multiplier_counter_textgenerationwebui">
+                                            </div>
+                                            <div data-tg-type="mancer, ooba, koboldcpp, aphrodite" data-newbie-hidden class="alignitemscenter flex-container flexFlowColumn flexBasis48p flexGrow flexShrink gap0">
+                                                <small data-i18n="Base">Base</small>
+                                                <input class="neo-range-slider" type="range" id="dry_base_textgenerationwebui" name="volume" min="1" max="4" step="0.01" />
+                                                <input class="neo-range-input" type="number" min="1" max="4" step="0.01" data-for="dry_base_textgenerationwebui" id="dry_base_counter_textgenerationwebui">
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="range-block">
+                                        <div class="range-block-title justifyLeft">
+                                            <span data-i18n="DRY Sequence Breakers">DRY Sequence Breakers</span>
```

Comment:
```suggestion
                                            <span data-i18n="Sequence Breakers">Sequence Breakers</span>
```

DRY is already in the title.

Reply:
Fixed.